projects/gopacket/fuzz_layer.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 Catena cyber All rights reserved.
```

Comment:
Please fix this.

Reply:
I will do a PR to the upstream repo as suggested by @Dor1s